src/styles/styles.scss

```diff
@@ -6,12 +6,16 @@
         so we don't interfere with other extension like reddit
 */
 
-@import url('./components/_grid.scss');
-@import url('./components/_dropdown.scss');
-@import url('./components/_card.scss');
-@import url('./components/_settings.scss');
-@import url('./components/_reload-modal.scss');
-@import url('./components/_add-snippet-modal.scss');
-@import url('./components/_readme-pages.scss');
-@import url('./components/_fixes.scss');
-@import url('./components/_import-export.scss');
+@use './components/prismjs-themes/prism-tomorrow.scss';
```

Comment:
Ahh, this works now? I guess we were just doing it wrong?

Reply:
When you use "import", sass does not add the contents of the imported file to the current file